wildfly/Dockerfile

```diff
@@ -1,31 +1,29 @@
 # Use latest jboss/wildfly image as the base
-FROM jboss/wildfly
+FROM jboss/wildfly:9.0.0.Final
 MAINTAINER Bruno Oliveira <bruno@abstractj.org>
 
 # Run everything below as root user
 USER root
 
-# Set the Java environment variable
-ENV JAVA_HOME /usr/lib/jvm/java-1.7.0-openjdk
-
 # Clean the metadata
-RUN yum -q clean all
+RUN yum install -y unzip && yum install -y wget && yum -q clean all
```

Comment:
Why two `yum` calls here? Install wget and unzip in sigle call for better build performance :)